jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import rospy
+from sensor_msgs.msg import Joy
+ 
+class JoyTopicCompletion:
```

Comment:
what is the role of this node? Does this try to move robot without pressing enable button? 
if so please add "README.md" page and show
- hardware setup
 - describe which harware (joystick) we're using, and how to install on the robot
 - add images, possibly animation gif, for experiments. something like https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=676343

Reply:
added README.md which contains
- Hardware description
- Demo gif